pkg/object/httpserver/metrics.go

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright (c) 2017, MegaEase
+ * All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package httpserver
+
+import (
+	"github.com/megaease/easegress/pkg/protocols/httpprot/httpstat"
+	"github.com/megaease/easegress/pkg/util/prometheushelper"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type (
+	metrics struct {
+		Health             *prometheus.GaugeVec
+		TotalRequests      *prometheus.CounterVec
+		TotalResponse      *prometheus.CounterVec
+		TotalErrorRequests *prometheus.CounterVec
+		RequestsDuration   prometheus.ObserverVec
+		RequestSizeBytes   prometheus.ObserverVec
+		ResponseSizeBytes  prometheus.ObserverVec
+	}
+)
+
+// newMetrics create the HttpServerMetrics.
+func (r *runtime) newMetrics(name string) *metrics {
```

Comment:
please put the content of this file to `runtime.go` and `mux.go` and remove this file.